public-ui/src/app/data-browser/data-browser.module.ts

```diff
@@ -0,0 +1,48 @@
+import {environment} from 'environments/environment';
```

Comment:
Where is all the data-browser code coming from? Is this from your branch you've been working on? Has this already been reviewed?

Reply:
Yeah. Redid what used to be in ui.